static/app/views/projectInstall/heartbeatFooter/index.tsx

```diff
@@ -0,0 +1,323 @@
+import {Fragment, useEffect} from 'react';
+import {RouteComponentProps} from 'react-router';
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import {openModal} from 'sentry/actionCreators/modal';
+import Button from 'sentry/components/button';
+import IdBadge from 'sentry/components/idBadge';
+import Placeholder from 'sentry/components/placeholder';
+import {IconCheckmark} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import pulsingIndicatorStyles from 'sentry/styles/pulsingIndicator';
+import space from 'sentry/styles/space';
+import {Project} from 'sentry/types';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
+
+import {useHeartbeat} from './useHeartbeat';
+
+enum BeatStatus {
+  AWAITING = 'awaiting',
+  PENDING = 'pending',
+  COMPLETE = 'complete',
+}
+
+async function openChangeRouteModal(
+  router: RouteComponentProps<{}, {}>['router'],
+  nextLocation: Location
+) {
+  const mod = await import(
+    'sentry/views/projectInstall/heartbeatFooter/changeRouteModal'
+  );
+  const {ChangeRouteModal} = mod;
+
+  openModal(deps => (
+    <ChangeRouteModal {...deps} router={router} nextLocation={nextLocation} />
+  ));
+}
+
+type Props = Pick<RouteComponentProps<{}, {}>, 'router' | 'route'> & {
+  issueStreamLink: string;
+  performanceOverviewLink: string;
+  projectSlug: Project['slug'];
+};
+
+export function HeartbeatFooter({
+  issueStreamLink,
+  performanceOverviewLink,
+  projectSlug,
+  router,
+  route,
+}: Props) {
+  const organization = useOrganization();
+
+  const {initiallyLoaded, fetchError, fetching, projects} = useProjects({
+    orgId: organization.id,
+    slugs: [projectSlug],
+  });
+
+  const projectsLoading = !initiallyLoaded && fetching;
+
+  const project =
+    !projectsLoading && !fetchError && projects.length ? projects[0] : undefined;
+
+  const {
+    sessionLoading,
+    eventLoading,
+    firstErrorReceived,
+    firstTransactionReceived,
+    sessionInProgress,
```

Comment:
`sessionInProgress`, `firstTransactionReceived` and  `firstErrorReceived ` fulfills all the 3 proposed options of the [Heartbeat RFC](https://github.com/getsentry/rfcs/blob/21098c90046db4247dd7046b51f1bf6ae832dfbf/text/0044-heartbeat.md#options-considered)